src/locales/en.json

```diff
@@ -288,9 +288,13 @@
     },
     "audio-categories": {
       "title": "Audio category",
+      "audiobook": "Audiobook",
       "music": "Music",
-      "sound": "Sound effects",
-      "podcast": "Podcast"
+      "news": "News",
+      "podcast": "Podcast",
+      "pronunciation": "Pronunciation",
+      "sound_effect": "Sound effects",
+      "sound": "Sound effects"
```

Comment:
This `sound` key is still required due to the API returning this value in some audios.

Reply:
Nice touch to display it here but not include it in the filters 👍